pkgs/development/python-modules/standard-mailcap/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  lib,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "standard_mailcap";
+  version = "3.13.0";
+  disabled = pythonOlder "3.13";
+  src = fetchPypi {
```

Comment:
```suggestion
  version = "3.13.0";
  pyproject = true;

  disabled = pythonOlder "3.13";

  src = fetchPypi {
```

Reply:
pyproject=true is in line 17. Did you miss it or do you want it further at the top?

Reply:
This is the style we usually follow.

Reply:
I reordered the attributes. Is the order/style (also the pipy vs github stuff) documented somewhere? Or is it just something you learn on the way?